config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -85,10 +88,11 @@ presubmits:
       path_alias: k8s.io/kubernetes
     spec:
       containers:
-        - image: gcr.io/k8s-testimages/kubekins-e2e:v20200329-0d2a077-experimental
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20200329-0d2a077-master
```

Comment:
We don't need the experimental image, we only enabled it in the past to use a newer version of bazel than k8s master was using and since we aren't using bazel locally anymore this isn't needed.